### ISSUE TITLE
508-defect-1❗️Launchblocker [SCREEN READER]: Lang attribute must be accurate to language of page/parts

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#22313]
# [508-defect-1 :exclamation: Launchblocker](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-1)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Josh

## Details

To meet Section 508 compliance, the lang attribute _must_ be accurate to the language on the page/parts. Spanish or tagalog screen reader users will be unable to understand most content on the page since it will be pronounced in an English accent.

Currently, both page wide language tags are inaccurate.

## Acceptance Criteria

- [ ] Lang attributes are accurate to page/parts on all translated pages for COVID-19 vaccines at VA

## Screenshots or Trace Logs
<img width="1085" alt="Screen Shot 2021-03-29 at 2 30 53 PM" src="https://user-images.githubusercontent.com/14154792/112883385-3b5a3e80-909c-11eb-9218-88acd5570a4c.png">
<img width="1277" alt="Screen Shot 2021-03-29 at 2 30 20 PM" src="https://user-images.githubusercontent.com/14154792/112883387-3bf2d500-909c-11eb-9f15-c2b7a5f66667.png">
<img width="1071" alt="Screen Shot 2021-03-29 at 2 28 01 PM" src="https://user-images.githubusercontent.com/14154792/112883389-3c8b6b80-909c-11eb-9bb3-b9cddabd377e.png">

**Comment by DanielleThierryUSDSVA** [comment]
HI @SKasimow @youngfreezyVA. Could you please review this ticket asap? Thank you!

**Comment by youngfreezyVA** [comment]
@joshkimux the only lang attributes that can be updated programmatically by the `i18Select` component are the `html` tags and the `div id=content`. looking at [your comment](https://github.com/department-of-veterans-affairs/vets-website/pull/16570#issuecomment-809700362) I think you are saying that all parts of the page need to have the lang attributes updated, including the side nav? that sounds out of scope for our release.

**Comment by youngfreezyVA** [comment]
for the breadcrumbs, the logic is determined by the url suffix:
```   <a 
          onclick="recordEvent({ 'event': 'nav-breadcrumb', 'breadcrumb-click-label': '{{ crumb.text }}', 'breadcrumb-click-level': '{{ forloop.index }}', 'breadcrumb-total-levels': '{{ crumbs.length }}' });" 
          aria-current="page" 
          href="/{{ path }}"
          lang="{{ entityUrl.path | detectLang }}"
          >    
            {{ crumb.text }}
          </a>````

**Comment by joshkimux** [comment]
@youngfreezyVA I think I may have a way forwards that can work to bump this down to a 508-defect-2. Let me know if this sounds fair!

There's three regions on this page that we need to address:
- the div with class="interior"
- the in this section component with links to each of the translated pages
- the i18-select links above the h1

Current state:

```html
<div class="interior" lang="en">
  <!-- In this section page links currently do not have lang attributes-->
  <nav>
    <a>COVID-19 vaccines at VA</a>
    <a>Vacunas contra COVID-19 en el VA</a>
    <a>Mga bakuna sa COVID-19 sa VA</a>
  </nav>
  <!-- i18-select -->
  <div class="i18-select">
    <a lang="en">English</a>
    <a lang="es">Espanol</a>
    <a lang="en">Tagalog</a>
  </div>
</div>
```

Start by correcting the tags in the i18 select for _all_ of the language pages, not just the English one
``` diff
<div class="interior" lang="en">
  <!-- In this section page links currently do not have lang attributes-->
  <nav>
    <a>COVID-19 vaccines at VA</a>
    <a>Vacunas contra COVID-19 en el VA</a>
    <a>Mga bakuna sa COVID-19 sa VA</a>
  </nav>
  <!-- i18-select -->
  <div class="i18-select">
    <a lang="en">English</a>
    <a lang="es">Espanol</a>
+    <a lang="tl">Tagalog</a>
  </div>
</div>
```


Add lang="en" to the In This Section nav and add specific lang attributes to the two non-English links

``` diff
<div class="interior" lang="en">
  <!-- In this section now has lang="en" with the specific language pages having their specific lang attributes -->
+  <nav lang="en">
     <a>COVID-19 vaccines at VA</a>
+    <a lang="es">Vacunas contra COVID-19 en el VA</a>
+    <a lang="tl">Mga bakuna sa COVID-19 sa VA</a>
  </nav>
  <!-- i18-select -->
  <div class="i18-select">
    <a lang="en">English</a>
    <a lang="es">Espanol</a>
    <a lang="tl">Tagalog</a>
  </div>
</div>
```

**Comment by SKasimow** [comment]
@youngfreezyVA can you point this and lmk if we can include in current sprint (ends April 6th) or should put in following sprint. Thanks!

**Comment by youngfreezyVA** [comment]
yes we can include in current sprint. per Josh's comment above we should break out the sidenav changes into a separate ticket, feel free to assign that one to me

**Comment by SKasimow** [comment]
@youngfreezyVA great - sprint 44 it is!
